tests/test_permission_to_get_one_is_supported.py

```
@@ -0,0 +1,78 @@
+    author = models.Author.objects.create(
+        name="Get Author",
+        birth_date=datetime.date(1990, 1, 1),
+        created_by=admin,
+    )
+
+    client.force_login(std_usr)
+    response = client.get(f"/api/gated-authors/{author.id}")
+    assert response.status_code == status.HTTP_404_NOT_FOUND, response.json()
+
+
+# TODO(phuongfi91): What about getting permitted resources that have unpermitted related resources?
```

Ruff (0.8.2)

Missing issue link on the line following this TODO

(TD003)